.azure-pipelines/merge.yml

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the ODPi Egeria project.
+---
+trigger:
+  branches:
+    include:
+      - master
+      - egeria-release-*
+pr: none
+
+stages:
+  - stage: Merge
+    jobs:
+      - job: Deploy
+        pool:
+          vmImage: 'ubuntu-latest'
+        steps:
+          - task: DownloadSecureFile@1
+            name: mavenSettings
+            inputs:
+              secureFile: egeria-artifactory-settings.xml
+          - script: |
+              mkdir -p $(Agent.HomeDirectory)/.m2/
+              mv $(mavenSettings.secureFilePath) $(Agent.HomeDirectory)/.m2/settings.xml
+          - task: ArtifactoryMaven@1
+            inputs:
+              mavenPomFile: 'pom.xml'
+              javaHomeOption: 'JDKVersion'
+              jdkVersionOption: '1.8'
+              jdkArchitectureOption: 'x64'
+              artifactoryResolverService: 'ODPiArtifactory'
+              targetResolveReleaseRepo: 'egeria-staging'
+              targetResolveSnapshotRepo: 'egeria-snapshot'
+              artifactoryDeployService: 'ODPiArtifactory'
+              targetDeployReleaseRepo: 'egeria-staging'
+              targetDeploySnapshotRepo: 'egeria-snapshot'
+              collectBuildInfo: true
+              buildName: 'Egeria'
+              buildNumber: '$(Build.BuildNumber)'
+              includeEnvVars: true
+              goals: 'clean install'
+              options: >-
```

Comment:
These days I'm using something like
```
mvn clean install -Dorg.slf4j.simpleLogger.showDateTime=true -Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss,SSS
```
to get timestamps, though perhaps we see that via pipelines anyway (need to check)

Reply:
The pipelines already give timestamps for each log line, and how long each step takes - I think they may be hidden by default though. If you click the three dots on a build and 'Toggle timestamps' they should show up.

Reply:
ok